src/main/java/org/folio/rest/impl/PostOrdersHelper.java

```diff
@@ -280,6 +295,114 @@ public PostOrdersHelper(HttpClientInterface httpClient, Map<String, String> okap
         throw new CompletionException(t.getCause());
       });
   }
+  
+  private CompletableFuture<Void> createReportingCodes(PoLine compPOL, JsonObject line,
+      List<ReportingCode> reportingCodes) {
+
+    List<String> reportingIds = new ArrayList<>();
+    List<CompletableFuture<Void>> futures = new ArrayList<>();
+    if(null!=reportingCodes)
+      reportingCodes
+      .forEach(reportingObject -> 
+        futures.add(createSubObjIfPresent(line, reportingObject, REPORTING_CODES, HelperUtils.getSubobjectapis().get(REPORTING_CODES))
```

Comment:
The endpoints specified in the `HelperUtils` have trailing slash however POST operations are expected against endpoints without it. As far as I know `okapi` on the discovery step will not be able to find module serving such path. Could you please check?

Reply:
Thanks Piotr